tecs/tecs.cpp

```diff
@@ -326,80 +298,81 @@ void TECS::_update_throttle_setpoint(const float throttle_cruise, const math::Ma
 	// Calculate the throttle demand
 	if (_underspeed_detected) {
 		// always use full throttle to recover from an underspeed condition
-		_throttle_setpoint = 1.0f;
+		_last_throttle_setpoint = 1.0f;
```

Comment:
This doesn't look correct as you're skipping the constrain and rate limit that was originally present below.

Reply:
That only happens in the non underspeed case.